nautobot/extras/templates/extras/objectchange_list.html

```diff
@@ -3,8 +3,9 @@
 
 {% block title %}Change Log{% endblock %}
 
-{% block sidebar %}
-    <div class="text-muted">
+{% block content %}
+    {{ block.super }}
+    <div class="text-muted pull-right">
         Change log retention:
```

Comment:
Feels like this could be better placed in the `header` block as a line just under the standard header contents, similar to what we do in object_retrieve.html for the "created / last updated" information?